src/lib/comms/include/sol-netctl.h

```diff
@@ -309,6 +375,146 @@ int32_t sol_netctl_service_get_strength(
     const struct sol_netctl_service *service);
 
 /**
+ * @brief Gets the service favorite
+ *
+ * This function gets the service favorite for the netctl service,
+ * after the service is to give by service monitor.
+ *
+ * @param service The service structure which the service favorite is
+ * desired
+ *
+ * @return 0-100 on success, -errno on failure.
+ */
+bool sol_netctl_service_get_favorite(
```

Comment:
It's quite difficult to understand what you trying to achieve here. Do you want to get the favorite service or this function tells if the service itself is the favorite? If this function is used to check if the service in question is a favorite service or not I think we should rename to  sol_netlctl_service_is_favorite()